guides/4.STYLING.md

```diff
@@ -0,0 +1,297 @@
+# Styling
+
+Terra Draw provides a number of ways to style the drawing experience. This guide will cover how to style features for the following Modes:
+
+- [Drawing Modes](#drawing-modes)
+- [Selection Mode](#selection-mode)
+- [Render Mode](#render-mode)
+
+## Drawing Modes
+
+Modes can be styled by passing a `styles` object to the constructor of the mode. For example, to style the `TerraDrawPolygonMode`:
+
+```javascript
+// Create Terra Draw
+const draw = new TerraDraw({
+  // Create Adapter
+  adapter: new TerraDrawMapboxGLAdapter({ map, lib }),
+
+  // Create Modes
+  modes: [
+    // Polygon Mode
+    new TerraDrawPolygonMode({
+      // Pass styles to the constructor
+      styles: {
+        // Fill colour (a string containing any CSS Legal Color Values, i.e. named, hex #, rgb(), rgba() etc.)
+        fillColor: "#00FFFF",
+
+        // Fill opacity (0 - 1)
+        fillOpacity: 0.7,
+
+        // Outline colour (CSS Legal Color Value)
+        outlineColor: "#00FF00",
+
+        //Outline width (Integer)
+        outlineWidth: 2,
+      },
+    }),
+  ],
+});
+```
+
+> [!TIP]
+> Colours can be any CSS Legal Color Values, e.g. named colors (`red`, `blue` etc.), hex values (`#00FFFF`), rgb/rgba values (`rgb(0,0,0)`, `rgba(255,255,255,0.5)`).
+
+Each Mode has a different set of styles that can be passed to the constructor.
+
+### Points
+
+The `TerraDrawPointMode` is styled using the following properties:
+
+| Property            | Type      | Example Value | Description                    |
+| ------------------- | --------- | ------------- | ------------------------------ |
+| `pointColor`        | CSS Color | `#00FFFF`     | The fill color of the point    |
```

Comment:
Colors in Terra Draw are modelled as 6 digit Hex colors (e.g. #00FFFF). If other color types (rgb, hsv, etc) 'work' this is mostly coincidental and I imagine we want to avoid having it in the documentation. This is just to ensure that we get consistency across providers.

Reply:
Updated and [added extra clarification](https://github.com/morehawes/terra-draw/blob/aec1d252f9b5eb21611067e6a31fcc93c7fbc578/guides/4.STYLING.md?plain=1#L44C1-L45C24) about this.